index.html

```diff
@@ -102,9 +102,213 @@ <h2>1989</h2>
 			</div>
 		</div>
 	</div>
+	<!-- END OF STORY -->
+
+	<!-- QUOTE -->
+	<div class="container-fluid" id="review">
+		<div class="row text-center">
+			<div class="container">
+				<h1>Fresno Bee</h1>
+				<p>"The freshest and best ingredients come together to make some of the best food I have ever ate. You can really taste the Central Valley in every bite"</p>
+			</div>
+		</div>
+	</div>
+	<!-- END QUOTE -->
+
+<div class="container-fluid" id="menu">
+	<div class="container">
+		<div class="row text-center">
+			<div class="col-md-12">
+				<h1>Menu</h1>
+			</div>
+		</div>
+		<div class="row">
```

Comment:
Mine reads: div class="row menu-tabs" Not sure if that was from class or something I added.